### ISSUE TITLE
v128 breaks support for SteelSeries (Sensei Ten)

### ISSUE THREAD

**Issue by cyablo** [Aurora-RGB/Aurora#138]
One of the  changes in v128 seem to break support for my SteelSeries Sensei Ten. The Mouse is detected by the the Plugin but doesn't respond on any changes and stays in default color.

**Comment by Aytackydln** [comment]
Are you using rgb.net implementation?

**Comment by cyablo** [comment]
Yes, worked before but suddenly stopped working with one of the latest Versions.

![steelseries](https://user-images.githubusercontent.com/3456720/221282940-57981b09-28ae-431a-a8bb-b4737ae4db02.JPG)

Mouse is detected but doesn't change lighting.

**Comment by Aytackydln** [comment]
Okay, I will need a little bit of help from you to find which version of rgb.net library broke it. Go to this nuget page:
https://www.nuget.org/packages/RGB.NET.Devices.SteelSeries/1.0.0

Choose a version and click "Download package" for .nupkg file. Open with zip or rar software, copy the dll in lib/ folder of archive to Aurora folder and start Aurora and see if it works.

**Comment by cyablo** [comment]
Im already down to 1.0.0-prerelease.92 but it's still not working. It  thing it might be somethingh else and not rgb.net.

**Comment by cyablo** [comment]
Even prelease.32 doesnt work, so it's surely something else that changed.

**Comment by Aytackydln** [comment]
Well, rgb.net's developer also said there wasn't any update about SteelSeries's sdk. Did you try with Aurora v127?

**Comment by Aytackydln** [comment]
reverted steelseries library change:
https://github.com/Aurora-RGB/Aurora/releases/tag/v130

**Comment by cyablo** [comment]
Sorry to say but I can't get SteelSerie RGB.NET implementation to work. No matter what Version of Aurora or Nuget's I use. Only GameSense works.

I already tried uninstalling SteelSeries GG software, uninstalling driver and using the one Windows 11 installs on default... Nope.

**Comment by cyablo** [comment]
Tried openrgb 0.8 from https://openrgb.org/releases.html, which works and can also control the mouse.

**Comment by cyablo** [comment]
That's strange: If I use the "Configure RGB.NET Devices" Dialog under "Devices & Wrappers", I can make the two LED's of the Sensei Ten blink white.

**Comment by Aytackydln** [comment]
Can you do clean install of Aurora?

**Comment by cyablo** [comment]
Did a clean Win 11 install on antother PC. Still not working with SteelSeries RGB.NET.

**Comment by cyablo** [comment]
To be honest: My SteelSeries Sensei Ten still does not work with OpenRGB. Even with v171. It got detected and says initialized, but mouse does not respond. SteelSeries GG integration however works.
